tests/rt.conf

```diff
@@ -304,114 +304,114 @@ RUN | atmaero_control_p8_rad_micro                      | - noaacloud
 COMPILE | atmaq | intel | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2 -D32BIT=ON | - jet     s4 | fv3 |
 RUN | regional_atmaq                                    | - jet s4                             | baseline |
 
-COMPILE | atmaq_debug | intel | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2 -DDEBUG=ON -D32BIT=ON | - jet noaacloud    gaea cheyenne s4 | fv3 |
-RUN | regional_atmaq_debug                              | - jet gaea cheyenne s4  noaacloud    | baseline |
+COMPILE | atmaq_debug | intel | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2 -DDEBUG=ON -D32BIT=ON | - jet noaacloud    gaea derecho s4 | fv3 |
```

Comment:
There are multiple tests which have been (for some reason) turned off on Cheyenne. Do we know these tests actually need to be turned off on Derecho also?

Reply:
atmaq cases run ok on derecho. no need to turn them off. clearly derecho bigger machine than cheyenne